crates/neon/src/types_docs.rs

```diff
@@ -0,0 +1,142 @@
+#[cfg_attr(aquamarine, aquamarine::aquamarine)]
```

Comment:
I think this should be `feature = "aquamarine"`.  This isn't currently being rendered in the docs.

https://deploy-preview-909--neon-bindings-docs.netlify.app/neon/types/index.html#the-javascript-type-hierarchy

Reply:
Great catch, thanks. Pushed a fix.